File: src/main/java/com/sandjelkovic/dispatchd/trakt/dto/EpisodeTrakt.java
package com.sandjelkovic.dispatchd.trakt.dto;


import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.RequiredArgsConstructor;

import java.time.Instant;
import java.util.HashMap;
import java.util.Map;

@Data
@RequiredArgsConstructor
@AllArgsConstructor
public class EpisodeTrakt {
	private String season;
	private Integer number;
	private String title;
	private Map<String, String> ids = new HashMap<>();
	private String overview;
	@JsonProperty("updated_at")
	private Instant updatedAt;
	@JsonProperty("first_aired")
	private Instant firstAired;

}

File: content-service/src/main/kotlin/com/sandjelkovic/dispatchd/contentservice/trakt/dto/EpisodeTrakt.kt
package com.sandjelkovic.dispatchd.contentservice.trakt.dto

import com.fasterxml.jackson.annotation.JsonProperty
import java.time.Instant
import java.time.ZonedDateTime

/**
 * @author sandjelkovic
 * @date 28.1.18.
 */
data class EpisodeTrakt(
        val season: String? = null,
        val number: Int? = null,
        val title: String? = null,
        val ids: Map<String, String> = mapOf(),
        val overview: String? = null,
        @JsonProperty("updated_at")
        val updatedAt: ZonedDateTime? = null,
        @JsonProperty("first_aired")
        val firstAired: Instant? = null
)
